app/templates/products/edit.html

```diff
@@ -0,0 +1,84 @@
+{% extends "base/base.html" %}
+{% block main_content %}
+    <div class="box box-primary">
+        <div class="box-header with-border">
+            <h3 class="box-title">Create Product</h3>
+        </div>
+        <!-- /.box-header -->
+        <!-- form start -->
+        <form role="form" method="post" enctype="multipart/form-data">
+            {{ form.hidden_tag() }}
+            <div class="box-body">
+                <div class="form-group">
+                    <label for="exampleInputEmail1">{{ form.name.label }}</label>
+                    {{ form.name(class="form-control")}}
+                    {% if  form.name.errors %}
+                        <ul class=errors>
```

Comment:
Macro byłoby jednak fajne, takie fantastyczne wyswietlajace pole i jesli sa to errorki :>